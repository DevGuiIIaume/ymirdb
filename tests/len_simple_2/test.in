set a1 1 2 3
set a2 -1 0
len a1
len a2
pop a1
len a1
push a1 0
len a1
list entries
append a1 4 5
len a1
pop a2
pop a2
len a2
pop a2
len a2
bye
