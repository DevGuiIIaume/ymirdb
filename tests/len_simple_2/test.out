> ok

> ok

> 3

> 2

> 1

> 2

> ok

> 3

> a2 [-1 0]
a1 [0 2 3]

> ok

> 5

> -1

> 0

> 0

> nil

> 0

> bye
